tslearn/docs/examples/classification/plot_shapelet_distances.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""
+Learning Shapelets: decision boundaries in 2D distance space
+============================================================
+
+This example illustrates the use of the "Learning Shapelets" method in order
+to learn a collection of shapelets that linearly separates the timeseries.
+In this example, we will extract two shapelets which are then used to
+transform our input time series in a two-dimensional space. Moreover, we
```

Comment:
may be add something like "in a two-dimensional space called the shapelet-transform space in the related literature."